language/stdlib/modules/libra_account.mvir

```diff
@@ -196,12 +207,71 @@ module LibraAccount {
         return;
     }
 
+    rotate_authentication_key_for_account(account: &mut Self.T, new_authentication_key: bytearray) {
+        *(&mut move(account).authentication_key) = move(new_authentication_key);
+        return;
+    }
+
     // Rotate the transaction sender's authentication key
     // The new key will be used for signing future transactions
     public rotate_authentication_key(new_authentication_key: bytearray) acquires T {
         let sender_account: &mut Self.T;
+
         sender_account = borrow_global_mut<T>(get_txn_sender());
-        *(&mut move(sender_account).authentication_key) = move(new_authentication_key);
+        if (*&copy(sender_account).delegated_key_rotation_capability) {
+            // The sender has delegated the privilege to rotate her key elsewhere--abort
+            abort(11);
```

Comment:
Good point; we need to do that in a follow-up. I don't want to add it to errors.rs right now because it will just end up being an unused constant, but we should document all of these soon.